src/app/components/grid/index.html

```diff
@@ -4,53 +4,116 @@
 >
 
   <sky-demo-page-summary>
-    This grid component displays a SKY UX-themed grid of data. The grid should contain <stache-code>sky-grid-column</stache-code> components that specify column information. The <stache-code>sky-grid</stache-code> component can provide a view of a searchable, paginated list component when used within the <a routerLink="../list-view-grid">sky-list</a> component.
+    This grid component displays a SKY UX-themed grid of data. The grid should contain <stache-code>sky-grid-column</stache-code> components that specify column information. The <stache-code>sky-grid</stache-code> component can provide a view of a searchable, paginated list component when used within <a routerLink="../list-view-grid">the <stache-code>sky-list</stache-code> component</a>.
   </sky-demo-page-summary>
 
   <sky-demo-page-properties sectionHeading="Grid component properties">
+    <sky-demo-page-property
+      propertyName="columns"
+    >
+    Specifies columns and column properties for the grid. This property accepts an array of <stache-code>SkyGridColumnModel</stache-code> values.
+    </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="data"
     >
-    Specifies an array of data for the grid. Each item in the array should have an ID and a property that maps to either the <stache-code>field</stache-code> or <stache-code>id</stache-code> property of each column in the grid.
+    Specifies the data for the grid. Each item requires an ID and a property that maps to the <stache-code>field</stache-code> or <stache-code>id</stache-code> property of each column in the grid. This property accepts an array of <stache-code>any</stache-code> values.
```

Comment:
> Each item requires an ID and a property...
Should this instead be:
> Each item requires an `id` and a property...
?

Reply:
Also, I always felt that the structure of `data` was pretty unclear until you scroll all the way down and look at the example below. Should we show a short example here? Or is that against our typical pattern?

Reply:
I assume it should be `id`. I'll make the change.

As for a short example, I think it would be great. What should the example look like?